### PR TITLE
Revert to using row.Scan instead of dbr.Load

### PR DESCRIPTION
**Pull request by wakamoleguy**
`dbr.Load` uses reflection to populate the results structs, which can run into issues when embedding multiple nested structs with overlapping field/column names. (It will overwrite the field in the first struct with the data that should be in the second struct, leaving the second struct zero'd out.) The original sqlc behavior used `row.Scan`, which instead looks at the returned columns in order. This would be tedious to enumerate by hand, but it doesn't matter when it's generated for us.

This seems orthogonal to the issue of using pointers vs `sql.NullString` etc for nullable types, and so I was able to largely undo our previous changes. The one change I did have to make was to preserve the behavior on `:one` queries when no record is found; we want to return nil instead of an error.

I tested this with my local server checkout:
1. Existing tests pass, which does cover some of the Schedules-related queries.
2. I was able to query Locations and Pinnables, embedded in the same query, and pass that data through to the frontend to confirm it was populating the name-overlapping fields correctly.

I'll also put up a draft PR of the server code changes for easier review of the changes generated. Edit: https://github.com/smartpass/smartpass-server/pull/151

### CODE REVIEW

File: internal/field.go
Comment (wakamoleguy) [inline internal/field.go:108]: This is unrelated to the other changes, but we had started manually configuring sqlc to rewrite `SchoolID` to `SchoolId`, etc. This changes the behavior at the source. (Note:  I could remove the `if` entirely, but it seemed clearer to keep it if we need to merge upstream changes.)

Comment (druvv) [reply internal/field.go:108]: Ah sweet. Thank you.

File: internal/field.go
Comment (druvv) [inline internal/field.go:104]: Did you try running the golang testsuite locally? Curious if there were any issues before we merge this.

**Review by druvv — APPROVED** [review]
lgtm!

File: internal/field.go
Comment (wakamoleguy) [reply internal/field.go:104]: No issues, although the coverage seems fairly low.

File: internal/field.go
Comment (wakamoleguy) [reply internal/field.go:108]: This didn't fully work, but I'll hold off on the rest of the changes for now. I see what needs to be done, it'll just be easier to review separately.

**Pull request merged**
